queries.sql

```diff
@@ -55,7 +55,7 @@ BEGIN;
 DELETE FROM animals;
 ROLLBACK;
```

Comment:
- [ ] There are some issues here, In the requirements there are four transactions that needs to be made in the way where you need to add all four transactions to the queries.sql file. but you are only adding two transactions to this file and even they are incomplete. because after every commit, update and rollback, you need to verify if that change has been made by querying and taking screenshots of that and also including these screenshots to your PR.
- [ ] Kindly add all four transactions to your queries.sql file, and also adding the screenshot of their results to your PR the way it is described in the requirements. kindly try to implement following requirements line by line.
requirements:
Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
Inside a transaction:
Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Verify that changes were made.
Commit the transaction.
Verify that changes persist after commit.
Take a screenshot of the results of your actions.
Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)
Take a screenshot of the results of your actions.
Inside a transaction:
Delete all animals born after Jan 1st, 2022.
Create a savepoint for the transaction.
Update all animals' weight to be their weight multiplied by -1.
Rollback to the savepoint
Update all animals' weights that are negative to be their weight multiplied by -1.
Commit transaction
Take a screenshot of the results of your actions.
Remember to add all these transactions to your queries.sql file.

Reply:
The transactions were implemented, but maybe not elaborate enough. Thank you for pointing this out. I have expanded the transactions attached images.